spec/features/bulk_updates/bulk_updates_data_centers_spec.rb

```diff
@@ -16,7 +16,7 @@
     before do
       # Search form
       select 'Entry Title', from: 'Search Field'
-      fill_in 'query_text', with: @find_and_remove_concept_response.body['EntryTitle']
+      find(:css, "input[id$='query_text']").set(@find_and_remove_concept_response.body['EntryTitle'])
```

Comment:
`fill_in` won't work because the ID field contains a timestamp. Easier for the tests to use the ending of the ID field and scope to ensure it is using the right field

Reply:
ok, yeah, that makes sense